templates/server/.mvn/maven.config

```diff
@@ -1,6 +1,5 @@
 #if($version)
-	#set ($versionArray = $version.split("-"))
-	-Drevision=$versionArray.get(0) -Dchangelist=-$versionArray.get(1)
+	-Drevision=$version
```

Comment:
fine but isnt `$version` always set? Then why the entire if and else?

Reply:
Why do we have this twice (`templates/server/.mvn/maven.config` and `templates/server/src/main/resources/archetype-resources/.mvn/maven.config`)?
IMHO this file is never used and can be deleted.